lib/reference/selectors.mjs

```diff
@@ -293,6 +294,10 @@ export const pseudoClasses = uniteSets(
 		'active-view-transition-type',
 		'any-link',
 		'autofill',
+		/*
+			introduced the 10/05/2016 by 8c808d0
+			as of 2024 it is still not supported by any browser
+		*/
 		'blank',
```

Comment:
Having both [`:blank`](https://developer.mozilla.org/en-US/docs/Web/CSS/:blank) and [`:blank`](https://drafts.csswg.org/css-page/#blank-pseudo)—line 219—could have been problematic if both were supported by at least one browser.
i.e. `['blank']` wouldn't differentiate between the two, which would be detrimental for developer

---

I am not removing it because #7510 hasn't been resolved yet.
i.e. removal policy has not been proposed
I have added its mozilla prefixed counterpart though.